### PR TITLE
Capstone Project - The Portal 🚀🎸

### PR DESCRIPTION
**Pull request by raminka13**
## With Great Projects Comes Great Responsibility 🕷🤘🏼

⌘ This has been a real challenge, to practice all code we learn in the past 4 weeks and deploy a functional website from scratch. It has been hard working days but we can be proud of the results and keep learning and improving in every aspect of this journey. ⌨️
- Add home first, section by section complete html-css. 🏠
- Finish home mobile version and start desktop version. 📱
- Add mobile menu html-css-js. 📝
- Add about page based on home. 📃
- Adjust navigation for both pages. 🧭
- Check linters for syntax or other code errors. ❌ ✅
- Record video showing how to explore the site. 📼
## [**DEMO VIDEO LINK**](https://youtu.be/_LSZU9BL6ek)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,812 @@
+:root {
```
Comment (elmar8287) [inline css/styles.css:1]: - [x] Please, consider using the original color. Here are the required ones:

#272A31, #EC5242, #D3D3D3

You are using only the last one, #D3D3D3.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    </section>
+
+    <section id="partners">
+        <h3 class="section-title">Partners</h3>
+        <div class="container">
+            <ul id="partners-list">
+                <li><a href="https://www.delltechnologies.com/" target="_blank" rel="noopener" class="partner"><img src="https://events.linuxfoundation.org/wp-content/uploads/2020/04/DellTech_Logo_Stk_Blue_Gry_rgb.svg" class="logo wp-post-image" alt=""></a></li>
+                <li><a target="_blank" rel="noopener" href="https://opensource.google/" class="partner"><img
+                            src="https://events.linuxfoundation.org/wp-content/uploads/google-logo-web.svg" alt=""></a>
+                </li>
```
Comment (elmar8287) [inline index.html]: - [x] I found, that the alternative text for the images is missing. Please, consider fixing it on the whale page.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
...88.3h-8.6V86.3z"/><path class="st0" d="M284.9,86.3h2.4V111h-2.4V86.3z"/><path class="st0" d="M302.7,85.8c7.7,0,11.6,6.1,11.6,12.9c0,6.8-3.9,12.9-11.6,12.9c-7.8,0-11.6-6.1-11.6-12.9 C291.1,91.9,295,85.8,302.7,85.8z M302.7,109.5c6.5,0,9.3-5.5,9.3-10.9c0-5.4-2.8-10.9-9.3-10.9c-6.5,0-9.3,5.5-9.3,10.9 C293.5,104.1,296.2,109.5,302.7,109.5z"/><path class="st0" d="M318,86.3h2.6l14.4,20.9h0.1V86.3h2.4V111h-2.6l-14.4-20.9h-0.1V111H318V86.3z"/> </g> <g> <path class="st0" d="M130.2,32.4h13.6v32.8h19.5v11.3h-33.1V32.4z"/><path class="st0" d="M172.3,32.4h13.6v44.1h-13.6V32.4z"/> <path class="st0" d="M196,32.4h13.9l12.9,23.6h0.1V32.4h12.9v44.1h-13.2L209,52.5h-0.1v24.1H196V32.4z"/><path class="st0" d="M286.1,59.5c0,12.3-6.5,18.1-20.1,18.1c-13.6,0-20.2-5.8-20.2-18.1V32.4h13.6v24.1c0,4.5-0.1,10.1,6.6,10.1 c6.4,0,6.4-5.7,6.4-10.1V32.4h13.6V59.5z"/><path class="st0" d="M307.7,53.1l-14.5-20.7h16l6.7,12l6.6-12h15.1l-14.2,20.8L339,76.6h-16.4l-7.5-13.2l-7.8,13.2h-15.6 L307.7,53.1z"/></g><g><polygon class="st1" points="22.2,88.8 22.2,44.4 0,44.4 0,111 66.6,111 66.6,88.8 "/> <polygon class="st0" points="111,0 0,0 0,33.3 22.2,33.3 22.2,22.3 88.8,22.3 88.8,88.8 77.7,88.8 77.7,111 111,111 "/></g></g></svg></a>
```
Comment (elmar8287) [inline about.html]: - [x] Please, consider that for About page is required to be without a Partners section and the Footer should be a dark color 👍

> Here is the original design for the About page

Screen Shot 2022-06-23 at 20 07 03

File: index.html [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+        </div>
+    </section>
+
+    <section id="program">
+        <h3 class="section-title">Main Program</h3>
+        <div class="program-container">
+            <div class="program-card">
+                <div id="lecture"></div>
+                <h4 class="progam-title">Lecture</h4>
+                <p class="program-text">Access leading experts to learn how to navigate the complex open source
```
Comment (elmar8287) [inline index.html:99]: - [x] Please, consider that in the Main program section you are missing the link for a whole program on the Desktop version and the button for the joining on the Mobile version. Please, look at the original required design:
> Desktop version
Screen Shot 2022-06-23 at 20 09 12

> Mobile version
Screen Shot 2022-06-23 at 20 08 47

File: index.html [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+                <p class="program-text">Find out what industry-leading companies and projects are doing in the future,
+                    and where technologies are headed</p>
+            </div>
+        </div>
+    </section>
+
+    <section id="speakers">
+        <h3 class="section-title dark">Featured Speakers</h3>
+        <div id="speaker-container"></div>
+    </section>
```
Comment (elmar8287) [inline index.html:132]: - [ ] Please, consider that on the Mobile version of the Home page, the Speakers section should display two speakers and the button More, as it is shown here in the original design:
Screen Shot 2022-06-23 at 20 14 01

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I want to highlight:

- You did a wonderful job! 👍 
- I liked the design and animations 👍 
- The PR description is perfect
- Also, the commit history is well 💯 👍 
- The README file is clear and professional 🥇 

## Required Changes ♻️

- [x] Check the comments under the review
- [ ] Your video presentation is perfect, but you have also to record something about the project code and especially highlight a piece of the code you find interesting or are proud of making. So just record the video again with mentioned info 👍 

## Optional suggestions

- Please, consider as an OPTIONAL suggestion to add the section in your README file, where describing how to setup your project on local machine (by using the `git clone` command) 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (raminka13) [reply index.html:132]: This is an optional requirement.
Not a core requirement.
I would like to learn how to implement this loading dynamic content function but is beyond my abilities right now.

File: README.md [hunk 6]
```diff
@@ -1,44 +1,38 @@
...
 - **HTML & CSS best practices:** Correct use of tags, elements, properties and syntax.
 - **GitHub flow:**  Correct use of Branches for deployment and features development.
-- **Linters Check:** Local and Pull Request check for errors, bugs and stylistic errors in code.
+- **Linters Check:** Local and Pull Request Github workflows to check for stylistic errors, bugs and syntax errors in code.
 
 ## Live Demo 
+[Github Pages Demo](https://raminka13.github.io/Capstone-M1/)
 
-[Live Demo Link](https://raminka13.github.io/M1-Portfolio/)
-
```
Comment (ShahierNashaat) [inline README.md:16]: - [x] Kindly add the presenting video link to the README file and as the previous code reviewer said that you should talk about the project code and especially highlight a piece of the code you find interesting or are proud of making.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+                <p class="program-text">Explore career opportunities with the world’s leading technology companies</p>
+            </div>
+            <div class="program-card">
+                <div id="future"></div>
+                <h4 class="progam-title">Future</h4>
+                <p class="program-text">Find out what industry-leading companies and projects are doing in the future,
+                    and where technologies are headed</p>
+            </div>
+        </div>
+        <a class="program-btn" href="#">See The Whole program</a>
```
Comment (ShahierNashaat) [inline index.html:125]: - [x] Kindly be noticed that the `See The Whole program` button should be under the cards as the design guideline shows.

Your Design | Design Guideline
---------------- | ------------------------
Screenshot from 2022-06-23 19-12-38 | screenshot

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @raminka13 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @raminka13     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**